picamera2/outputs/ffmpegoutput.py

```diff
@@ -70,7 +71,8 @@ def start(self):
                            '-thread_queue_size', '1024',  # necessary to prevent warnings
                            '-i', self.audio_device]
             audio_codec = ['-b:a', str(self.audio_bitrate),
-                           '-c:a', self.audio_codec]
+                           '-c:a', self.audio_codec,
+                           -'af', self.audio_filter]
```

Comment:
Before I merge this, just one thing I wanted to understand. We're now setting the audio filter to `pan=stereo` by default. Is this a change in behaviour, or was it doing the same thing before? (If it is a change, do you know what you would put to get the previous behaviour?) Thanks!

Reply:
Good call, I intended this to be a sane default, but looking into it a bit more and if the input or soundcard is configured for mono, this would indeed change it to stereo (the `-af` default will reflect the input signal/channel setup if nothing is provided to that argument). Its quite rare that a device nowadays would be mono, but im sure there are use-cases for that. 

So I went ahead and removed the default parameter to `pan=stereo` so it will have the same behavior as before, while allowing it to be customized if an arg is passed.

Reply:
I think we've still got the problem where `audio_filter` is a required parameter, and causing a test failure. Could we put it back where it was (i.e. just before `pts`, just in case anyone out there has hard-coded the order of all those audio parameters) and default it to `None`. Then I think the code you've got after that looks good.

Reply:
done 💯